scrapers/scraper.py

```
@@ -0,0 +1,424 @@
+    "Gammattor\n\n\n\nTower of Zero Disturbances": "Gammattor 1111\n\n\n\nTower of Zero Disturbances",
+    " (officially released)": "",
+    " (nerf)": "",
+    "April Fools 2019": ""
+}
+
+def get_tower_abbreviation(x):
+    return ''.join([word[0].lower() if word[0].islower() else word[0].upper() for word in x.replace('-', ' ').split()]).lower()
+
+def open_files(x):
+    global file, file2, info_file
+    file = open(f"public/static/data/games/{x}.txt", "w")
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)